spec/models/doorkeeper/access_token_spec.rb

```diff
@@ -12,6 +12,70 @@ module Doorkeeper
       let(:factory_name) { :access_token }
     end
 
+    describe :generate_token do
+      it 'generates a token using the default method' do
+        FactoryGirl.create :access_token
+
+        token = FactoryGirl.create :access_token
+        expect(token.token).to be_a(String)
+      end
+
+      it 'generates a token using a custome class' do
+        module CustomGenerator
+          def self.generate(opts={})
```

Comment:
Unused method argument - `opts`. If it's necessary, use `_` or `_opts` as an argument name to indicate that it won't be used. You can also write as `generate(*)` if you want the method to accept any arguments but don't care about them.<br>Surrounding space missing in default value assignment.